migrations/v5.js

```diff
@@ -0,0 +1,154 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('List - v5.1.0 to v5.2.0', async () => {
+  let lists;
+  whereFromPlugin('List - from v5.1.0', { name: 'adapt-list', version: '<5.2.0' });
+  whereContent('where content 1', async content => {
+    lists = content.filter(({ _component }) => _component === 'list');
+    return lists.length;
+  });
+  mutateContent('List - add _graphic object', async content => {
+    lists.forEach(list => {
+      list._items.forEach(item => { if (!_.has(item, '_graphic')) _.set(item, '_graphic', {}); });
```

Comment:
```suggestion
      list._items.forEach(item => { 
        if (!_.has(item, '_graphic')) _.set(item, '_graphic', {}); 
      });
```
That's a lot to read on one line